woocommerce/woocommerce
# Product Collection: Fix max price filter query value inclusion

Partial work for #50019

### Changes proposed in this Pull Request:

The max price filter query is set to only include products that cost less than the given max value. This PR fixes that by changing the query to less **or equal** to the max price filter value.

### How to test the changes in this Pull Request:

1. Add Product Collection block, choose `create your own`
2. Add Product Filters pattern
3. Save and go to frontend
4. Filter by the max price that is equal to the price of one of the products and ensure that the product is included in the results.
